### PR TITLE
added template for push

### PR DESCRIPTION
**Pull request by edewit**

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -55,6 +55,49 @@
...
+      "description": "An example Project using our Unified Push Server",
+      "type": "default",
+      "icon": "icon-refresh",
+      "category": "Samples",
+      "screenshots": [],
+      "appTemplates": [{
+        "id": "ups_cloud",
+        "name": "PUsh Cloud App",
+        "type": "cloud_nodejs",
+        "repoUrl": "git@redhat-demos-t.sandbox.feedhenry.com:redhat-demos-t/Push-Hello-World-Cloud-App.git",
```
Comment (johnfriz) [inline global.json]: Move this over to https://github.com/feedhenry-templates

File: global.json [hunk 2]
```diff
@@ -55,6 +55,49 @@
...
+        "id": "push_app",
+        "name": "Push Cordova Light Client",
+        "type": "client_hybrid",
+        "repoUrl": "git@redhat-demos-t.sandbox.feedhenry.com:redhat-demos-t/Push-Hello-World-Cordova-Light-Client.git",
+        "githubUrl": "https://redhat-demos-t.sandbox.feedhenry.com:redhat-demos-t/Push-Hello-World-Cordova-Light-Client.git",
+        "repoBranch": "refs/heads/master",
+        "icon": "icon-html5",
+        "category": "Sample Apps",
+        "screenshots": [{
+          "url": "https://raw.githubusercontent.com/edewit/aerogear.org/feedhery-blog/img/news/2015-04-20-feedhenry-push-templates/9.png"
```
Comment (johnfriz) [inline global.json]: Add the screenshots to the template-config repo.

**Comment by edewit** [comment]
Changed the references, but I can't move the projects to https://github.com/feedhenry-templates

File: global.json [hunk 3]
```diff
@@ -55,6 +55,49 @@
...
+        "id": "push_console",
+        "name": "Push Console App",
+        "type": "webapp_advanced",
+        "repoUrl": "git://github.com/feedhenry-templates/Push-Hello-World-Console.git",
+        "githubUrl": "https://github.com/feedhenry-templates/Push-Hello-World-Console.git",
+        "repoBranch": "refs/heads/master",
+        "icon": "icon-nodejs",
+        "category": "Sample Apps",
+        "screenshots": [
+          "url": "https://raw.githubusercontent.com/feedhenry/template-config/master/screenshots/apps/push_hello_world/console.png"
```
Comment (cianclarke) [inline global.json]: Invalid JSON?

File: global.json [hunk 4]
```diff
@@ -55,6 +55,49 @@
         "screenshots": []
       }]
     }, {
+      "id": "ups_project",
+      "priority": 0.410,
+      "name": "Push Hello World",
+      "description": "An example Project using our Unified Push Server",
+      "type": "default",
+      "icon": "icon-refresh",
```
Comment (cianclarke) [inline global.json]: Would like to find something more appropriate here in glyphshish, we're already using icon-refresh. inspiration here http://fortawesome.github.io/Font-Awesome/3.2.1/icons/

File: global.json [hunk 5]
```diff
@@ -55,6 +55,49 @@
...
+      "category": "Samples",
+      "screenshots": [],
+      "appTemplates": [{
+        "id": "ups_cloud",
+        "name": "PUsh Cloud App",
```
Comment (cianclarke) [inline global.json]: PUsh -> Push

File: global.json [hunk 6]
```diff
@@ -55,6 +55,49 @@
...
+        "type": "cloud_nodejs",
+        "repoUrl": "git://github.com/feedhenry-templates/Push-Hello-World-Cloud-App.git",
+        "githubUrl": "https://github.com/feedhenry-templates/Push-Hello-World-Cloud-App.git",
+        "repoBranch": "refs/heads/master",
+        "icon": "icon-cloud",
+        "category": "Sample Apps"
+      }, {
+        "id": "push_app",
+        "name": "Push Cordova Light Client",
```
Comment (cianclarke) [inline global.json]: "Push Notifications Mobile Client" might be better

**Comment by cianclarke** [comment]
Hi @edewit - can you either add me as a contributor to your fork, or update the icon to `icon-plane` (or another of your choosing)? 
Then, I can get this merged back to master!
Cheers

**Comment by cianclarke** [comment]
Thanks @edewit

**Pull request merged**